services/QuillLMS/app/controllers/integrations_controller.rb

```diff
@@ -0,0 +1,131 @@
+class IntegrationsController < ApplicationController
+  include HTTParty
+  layout "integrations"
+
+  def amplify
+    store_partner_session()
```

Comment:
we don't really have a style guide so I don't know if we want to care about this, but since this is Ruby the empty parentheses are not necessary